ui/raidboss/data/triggers/seiryu-ex.js

```diff
@@ -133,20 +133,66 @@
         en: 'Get Tower',
       },
     },
+    {
+      id: 'SeiryuEx Handprint East Move',
+      regex: / 15:\y{ObjectId}:Yama-no-shiki:37E5:Handprint:/,
+      regexDe: / 15:\y{ObjectId}:Yama no Shiki:37E5:Handabdruck:/,
+      condition: function(data) {
+        return data.firstHandprint !== undefined;
+      },
+      infoText: {
+        en: 'Move, East',
+      },
+    },
+    {
+      id: 'SeiryuEx Handprint West Move',
+      regex: / 15:\y{ObjectId}:Yama-no-shiki:37E6:Handprint:/,
+      regexDe: / 15:\y{ObjectId}:Yama no Shiki:37E6:Handabdruck:/,
+      condition: function(data) {
+        return data.firstHandprint !== undefined;
+      },
+      infoText: {
+        en: 'Move, West',
+      },
+    },
     {
       id: 'SeiryuEx Handprint East',
       regex: / 15:\y{ObjectId}:Yama-no-shiki:37E5:Handprint:/,
       regexDe: / 15:\y{ObjectId}:Yama no Shiki:37E5:Handabdruck:/,
+      condition: function(data) {
+        return data.firstHandprint === undefined;
+      },
       infoText: {
-        en: 'East =>',
+        en: 'East',
+      },
+      run: function(data) {
+        data.firstHandprint = true;
       },
     },
     {
       id: 'SeiryuEx Handprint West',
       regex: / 15:\y{ObjectId}:Yama-no-shiki:37E6:Handprint:/,
       regexDe: / 15:\y{ObjectId}:Yama no Shiki:37E6:Handabdruck:/,
+      condition: function(data) {
+        return data.firstHandprint === undefined;
+      },
       infoText: {
-        en: '<= West',
+        en: 'West',
+      },
+      run: function(data) {
+        data.firstHandprint = true;
+      },
+    },
+    {
+      id: 'SeiryuEx Force of Nature',
+      regex: /Yama-no-shiki:37E9:/,
+      delaySeconds: 1,
+      infoText: {
+        en: 'Out of middle',
```

Comment:
This seems good! An alternative might be to take "go to snakes" callout and make the ones after the first time say something like "out of middle toward snakes" and combine these two?

Or do you think it's better to have both?